SPM Sample/SPM-sample-app.xcodeproj/project.pbxproj

```diff
@@ -526,7 +526,7 @@
 			repositoryURL = "https://github.com/ContentSquare/CS_iOS_SDK.git";
 			requirement = {
 				kind = upToNextMajorVersion;
-				minimumVersion = 4.22.1;
+				minimumVersion = 4.23.0;
```

Comment:
nit: I still don't agree with having the minimum version as the latest version while having `upToNextMajorVersion`

Reply:
Yeah I agree with you, shall we just set it to `Exact` ?

Reply:
yes sir, exact would make sense in this case as we change it manually every release but it is a Nit, we can ignore my comment if it doesn't make that much sense here

[some background story](https://contentsquare.slack.com/archives/C02HU09E0JG/p1683880402946659)

Reply:
set to exact